src/libraries/Microsoft.Extensions.Configuration.Abstractions/ref/Microsoft.Extensions.Configuration.Abstractions.cs

```diff
@@ -6,12 +6,22 @@
 
 namespace Microsoft.Extensions.Configuration
 {
+    public readonly partial struct ConfigurationDebugViewContext
```

Comment:
The new APIs here:
- struct ConfigurationDebugViewContext
- ConfigurationExtensions.Exists
- ConfigurationRootExtensions.GetDebugView

do not exist in the approved API @ https://github.com/dotnet/runtime/issues/85486#issuecomment-1568858340

Reply:
These aren't new APIs. This is just taking the output of `/t:GenerateReferenceAssemblySource` so the next person doesn't need to fight it.

Reply:
CC @ViktorHofer @ericstj

Reply:
ConfigurationDebugViewContext just moved to be ordered correctly, ConfigurationExtensions.Exists added Attribute suffix on NotNullWhenAttribute and ConfigurationRootExtensions.GetDebugView included the full namespaces.

Probably someone was hand-editing this file before to cause these diffs.  The changes look fine.